components/progress-bar/src/ProgressBar.js

```diff
@@ -0,0 +1,110 @@
+/** @jsx jsx */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { jsx, useTheme } from '@westpac/core';
+
+import { SrOnly } from '../../accessibility-helpers/src';
+
+// ==============================
+// Utils
+// ==============================
+
+// ==============================
+// Component
+// ==============================
+
+/**
+ * Progress Bar: A visual indication of progress. Use when loading content or to indicate how far along the user is in a journey.
+ */
+export const ProgressBar = ({ valueNow, skinny, ...props }) => {
+	const { progressBar } = useTheme();
+
+	// Common styling
+	const common = {
+		height: skinny ? progressBar.size.skinny.height : progressBar.size.default.height,
+		marginBottom: progressBar.marginBottom,
+		overflow: 'hidden',
+		backgroundColor: 'white',
+		borderRadius: skinny ? progressBar.size.skinny.height : progressBar.size.default.height,
+		border: `${progressBar.borderWidth} solid`,
+		borderColor: progressBar.borderColor,
+		position: 'relative',
+
+		'::after': {
+			content: '"0%"',
+			position: 'absolute',
+			left: '10px',
```

Comment:
should have a top and height too so it's height/position matches the bar (missing in GUI2.0)

top: '1px',
height: '100%',

Reply:
update top and left to rems once Core is merged